lib/crons.ex

```diff
@@ -0,0 +1,132 @@
+defmodule ApplicationRunner.Crons do
+  @moduledoc """
+    ApplicationRunner.Crons delegates methods to the corresponding service.
+  """
+
+  import Ecto.Query, only: [from: 2, from: 1]
+
+  alias ApplicationRunner.Crons.Cron
+  alias ApplicationRunner.Environment
+  alias ApplicationRunner.Errors.BusinessError
+  alias ApplicationRunner.Errors.TechnicalError
+  alias ApplicationRunner.EventHandler
+  alias ApplicationRunner.Repo
+  alias Crontab.CronExpression.{Composer, Parser}
+
+  def run_cron(
+        action,
+        props,
+        event,
+        env_id
+      ) do
+    with {:ok, metadata} <-
+           Environment.create_metadata(env_id),
+         {:ok, _pid} <-
+           Environment.ensure_env_started(metadata) do
+      EventHandler.send_env_event(env_id, action, props, event)
+    end
+  end
+
+  def create(env_id, %{"listener_name" => _action} = params) do
+    cron =
+      env_id
+      |> Cron.new(params)
+      |> Ecto.Changeset.apply_changes()
+
+    cron
+    |> to_job()
+    |> ApplicationRunner.Scheduler.add_job()
+
+    cron.name
+  end
+
+  def create(_env_id, params) do
+    BusinessError.invalid_params_tuple(params)
+  end
+
+  def get(id) do
+    case Repo.get(Cron, id) do
+      nil -> TechnicalError.error_404_tuple(id)
```

Comment:
The identifier is not in the url so we probably want to return an error other than 404.